msg/Gpgga.msg

```diff
@@ -21,5 +21,5 @@ string altitude_units
 
 float32 undulation
 string undulation_units
-uint32 diff_age
+float64 diff_age
```

Comment:
This would be a breaking change, so it can't be released in any current distros, but we could release it in Rolling and ROS 2 J-turtle.

Also, from the references you list, the maximum precision of this field is 1/10 second, so 32-bit float should be sufficient.

Reply:
I agree, I was very concerned with this commit too. In my application, it has to be float, but the 1/10 second can be easily dropped for now. The device I'm using is an EMLID Reach M2.